package.json

```diff
@@ -3,19 +3,23 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "dev": "next dev",
-    "dev-ssl": "npm-run-all --parallel ssl-proxy dev",
+    "dev": "run-p ssl-proxy dev:insecure",
+    "dev:insecure": "next dev",
     "ssl-proxy": "local-ssl-proxy --source 4000 --target 3000",
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "prisma": "dotenv -c development prisma",
-    "test": "run-s lint testUnit testIntegration",
-    "testUnit": "jest",
-    "testIntegration": "start-server-and-test dev-ssl http://localhost:3000 cypress:headless",
-    "tdd": "jest --watch",
-    "cypress": "cypress open",
-    "cypress:headless": "cypress run"
+    "prisma": "dotenv -c development -- prisma",
+    "prisma:test": "dotenv -c test -- prisma",
+    "test": "run-s lint test:unit:run test:e2e:run",
+    "test:unit:run": "jest",
+    "test:unit:dev": "jest --watch",
+    "test:e2e:run": "cross-env NODE_ENV=test start-server-and-test 'run-s docker:up dev' :3000 'run-s \"prisma:test -- migrate reset --force\" cy:run docker:down'",
+    "test:e2e:dev": "cross-env NODE_ENV=test start-server-and-test 'run-s docker:up dev' :3000 'run-s \"prisma:test -- migrate reset --force\" cy:dev docker:down'",
```

Comment:
This doesn't work on Windows. I am going to try it on Linux, but it will take me a while to check, since I usually use `podman` and `podman-compose` as alternatives to Docker on Linux (Docker doesn't play nice with newer Linux kernels).

Reply:
If Docker doesn't work on Linux where would it work 😦

Reply:
It's just a bit of a pain to set up Docker on Fedora. It can mess up virtualized networks. As well, a lot of the k8s infrastructure doesn't use Docker. Managed to get it working, though.

Reply:
Ahh okay. Thanks for getting it to work! The latest commit should fix the scripts on Windows.